virttest/qemu_qtree.py

```diff
@@ -163,15 +163,7 @@ def add_child(self, child):
         super(QtreeDev, self).add_child(child)
 
     def guess_type(self):
-        if self.qtree['type'] == 'virtio-blk-device':
-            return QtreeDisk
-        elif ('drive' in self.qtree and
-              self.qtree['type'] != 'usb-storage' and
-              self.qtree['type'] != 'virtio-blk-pci'):
```

Comment:
@spcui,

I think this logical looks more clearly,  expect for usb-strorage, other devices if it has backend driver, it should be a disk.

if 'driver' in self.qtree and self.qtree['type'] != 'usb-storage':
    return QtreeDisk
else:
    return QtreeDev